index.md

```diff
@@ -17,29 +17,29 @@ and main developer is [David
 Aspinall](http://homepages.inf.ed.ac.uk/da). Other contributors are
 listed below and in the [AUTHORS](/AUTHORS) file.
 
-Proof General comes ready-to-go for these proof assistants:
+Proof General offers a full support for latest versions of the Coq proof assistant:
 
 - [![Coq logo](img/coq.png)](https://coq.inria.fr/)
-  **Coq Proof General** for [Coq](https://coq.inria.fr/)  
+  **Coq Proof General**,
+  for [Coq](https://coq.inria.fr/)  
   by Healfdene Goguen, Patrick Loiseleur, David Aspinall, and [Pierre Courtieu](http://cedric.cnam.fr/~courtiep/).
 
+There are also instances of Proof General that can handle the following proof assistants:
```

Comment:
Maybe "Proof General also supports previous versions of the following other proof assistants"?